src/notification-banner/private/NotificationBannerView.module.scss

```diff
@@ -0,0 +1,167 @@
+// Themes
```

Comment:
This is a port of the existing terra-alert stylesheet with minor (if any) changes